website/static/js/filepage/index.js

```diff
@@ -87,9 +87,35 @@ var FileViewPage = {
         self.file = self.context.file;
         self.node = self.context.node;
         self.editorMeta = self.context.editor;
-        //Force canEdit into a bool
-        self.canEdit = m.prop(!!self.context.currentUser.canEdit);
-
+        self.file.checkout_user = null;
+        self.request_done = false;
+        self.isCheckout_user = function() {
+            $.ajax({
+                method: 'get',
+                url: window.contextVars.apiV2Prefix + 'files' + self.file.path + '/',
+                beforeSend: $osf.setXHRAuthorization
+            }).done(function(resp) {
+                self.request_done = true;
+                self.file.checkout_user = resp.data.relationships.checkout.links.related ? ((resp.data.relationships.checkout.links.related.href).split('users/')[1]).replace('/', ''): null;
```

Comment:
Can you handle the case where `resp.data.relationships.checkout.links.related.href` is `null`? I'm getting an `Uncaught TypeError: Cannot read property 'split' of null` when I check-out a file and then force check-in as a different user.